synthesizer/program/src/logic/command/await_.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::Opcode;
+use console::{network::prelude::*, program::Register};
+
+/// An await command, e.g. `await r0;`.
```

Comment:
Why does this file end in `_.rs`? It also happens in some other await files. Haven't seen that before in this codebase

Reply:
presumably due to imports, as `await` is a reserved keyword

Reply:
This is normal